app/mainwindow.cpp

```diff
@@ -480,14 +479,12 @@ void MainWindow::lastUsedTheme()
     currentTheme = themeCollection->theme(themeName);
     applyCurrentTheme();
 
-    for (auto action : stylesGroup->actions()) {
-        if (action->text() == themeName) {
-            action->setChecked(true);
-            break;
-        }
-    }
-
-    styleLabel->setText(themeName);
+	foreach(QAction* a, stylesGroup->actions()) {
```

Comment:
Any specific reason for this change? C++11 is activated in app.pro. I think that is fine nowadays.